tools/pd-control.md

```diff
@@ -365,26 +365,28 @@ Success!
 
 ### operator [show | add | remove]
 
-用于显示和控制调度操作。
+用于显示和控制调度操作，或者分裂一个 region。
```

Comment:
分裂一个region也是属于调度操作，没必要单独提出来

Reply:
Someone from VLDB asked us if applications can customize the boundary of a region. I think it's  okay to separate this from scheduling because it is too general.

Reply:
那merge是否有必要也提出来，而且`或者`会让人觉得分裂与显示和控制是平级的，但实际分裂是控制的一种，用`包括`更好一些

Reply:
嗯，好，Merge 也提出来吧。把具体能做的操作拆分一下